### PR TITLE
Fix hvh comms

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Give rebel faction proper comms.

Also remove some station bounced radio, because they are able to listen to the loyalist common channel. 

## Why It's Good For The Game

Proper communication

## Changelog
:cl:
fix: Rebel have their own comms
/:cl:

### CODE REVIEW

File: code/game/objects/items/radio/electropack.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 	if(href_list["freq"])
 		SSradio.remove_object(src, frequency)
-		frequency = sanitize_frequency(frequency + text2num(href_list["freq"]))
+		frequency = sanitize_frequency(frequency)
```
Comment (BraveMole) [inline code/game/objects/items/radio/electropack.dm:44]:
```suggestion
		frequency = sanitize_frequency(frequency + text2num(href_list["freq"]))
```

File: code/game/objects/items/radio/headset.dm [hunk 2]
```diff
@@ -121,6 +121,8 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 /obj/item/radio/headset/talk_into(mob/living/M, message, channel, list/spans, datum/language/language)
 	if(!listening)
 		return ITALICS | REDUCE_RANGE
+	if(channel && frequency == FREQ_COMMON_REBEL)
+		channel += " Rebel"
```
Comment (BraveMole) [inline code/game/objects/items/radio/headset.dm:125]: i am not even ashamed by this

Comment (TiviPlus) [reply code/game/objects/items/radio/headset.dm:125]: You should be, Why is it like this

File: code/__HELPERS/unsorted.dm [hunk 3]
```diff
@@ -236,12 +236,9 @@ GLOBAL_REAL_VAR(list/stack_trace_storage)
 
 
 // Ensure the frequency is within bounds of what it should be sending/receiving at
-/proc/sanitize_frequency(frequency, free = FALSE)
+/proc/sanitize_frequency(frequency)
 	. = round(frequency)
-	if(free)
-		. = clamp(frequency, MIN_FREE_FREQ, MAX_FREE_FREQ)
-	else
-		. = clamp(frequency, MIN_FREQ, MAX_FREQ)
```
Comment (TiviPlus) [inline code/__HELPERS/unsorted.dm:244]: ???

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]
This is better

**Pull request merged**